pymatgen/io/vasp/inputs.py

```diff
@@ -1112,11 +1114,11 @@ def __init__(
                 (or negative), VASP automatically generates the KPOINTS.
             style: Style for generating KPOINTS. Use one of the
                 Kpoints.supported_modes enum types.
-            kpts (2D array): 2D array of kpoints. Even when only a single
+            kpts (2D array): Array of kpoints. Even when only a single
                 specification is required, e.g. in the automatic scheme,
                 the kpts should still be specified as a 2D array. e.g.,
-                [[20]] or [[2,2,2]].
-            kpts_shift (3x1 array): Shift for Kpoints.
+                [(20,),] or [(2, 2, 2),].
+            kpts_shift (3x1 array): Shift for kpoints.
             kpts_weights: Optional weights for kpoints. Weights should be
```

Comment:
This docstring differs (type of list[int]) from the implementation: https://github.com/materialsproject/pymatgen/blob/ff94faa5bb490354c420f188f1abeac17cb40931/pymatgen/io/vasp/inputs.py#L1524